on-demand system now using NO_PROXY/no_proxy env (#174)
KxSystems/docs: docs/learn/licensing.md

'2018.03.28T11:20:03.831 couldn't connect to license daemon -- exiting
```

> If a HTTP proxy is required, the environment variables `http_proxy` or `HTTP_PROXY` define the URL of the HTTP proxy to use.
> Since 4.1t 2022.11.01,4.0 2022.10.26,4.0ce 2022.09.16 the on-demand system now honours the NO_PROXY/no_proxy environment variables, with the lowercase version taking precedence.

Reviewer: ```suggestion
If an HTTP proxy is required, the environment variables `http_proxy` or `HTTP_PROXY` define the URL of the HTTP proxy to use.
Since 4.1t 2022.11.01,4.0 2022.10.26,4.0ce 2022.09.16 the on-demand system honours the NO_PROXY/no_proxy environment variables, with the lowercase version taking precedence.
```
